components/sup/src/manager/service/mod.rs

```diff
@@ -771,8 +786,13 @@ impl Service {
     /// Helper for compiling configuration templates into configuration files.
     ///
     /// Returns `true` if the configuration has changed.
-    fn compile_configuration(&self, ctx: &RenderContext<'_>) -> bool {
-        match self.config_renderer.compile(&self.pkg, ctx) {
+    fn compile_configuration(&self, ctx: &RenderContext) -> bool {
+        match self.config_renderer.compile(
+            &ctx.service_group_name(),
+            &self.pkg,
+            &self.pkg.svc_config_path,
+            ctx,
```

Comment:
Right. This adds:

* `service_group_name` as a separate parameter. `compile` used to just grab that from the `ctx` which is what we are passing here. The reason why this is added separately is that if you were compiling something outside of a service context, there would not be a `service_group`. `compile` basically just uses this string as a logging prefix and in the case of an `install` hook we just pass in the package name.
* the path where the template will be rendered. This will differ depending on whether you are compiling to `config` or `config_install`